packages/react-component-library/src/components/TabSet/TabSet.stories.tsx

```diff
@@ -2,25 +2,21 @@ import React from 'react'
 import { storiesOf } from '@storybook/react'
 
 import TabSet from './index'
-
-const tabData = [
-  {
-    title: 'Example Tab 1',
-    content: 'This is some example tab 1 content',
-  },
-  {
-    title: 'Example Tab 2',
-    content: 'This is some example tab 2 content',
-  },
-]
+import Tab from './tab'
```

Comment:
why not call it ./Tab

I know it's petty, just reminding myself we need to get a consistent naming convention.